decidim-bulletin_board-app/spec/mutations/publish_results_mutation_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Mutations
+  RSpec.describe PublishResultsMutation, type: :request do
+    subject { post "/api", params: { query: query, variables: { messageId: message_id, signedData: signed_data } }, headers: headers }
+
+    let(:query) do
+      <<~GQL
+        mutation($messageId: String!, $signedData: String!) {
+          publishResults(messageId: $messageId, signedData: $signedData) {
+            election {
+              id
+              status
+            }
+            error
+          }
+        }
+      GQL
+    end
+
+    let!(:election) { create(:election, status: :results) }
+    let(:authority) { Authority.first }
+    let(:headers) { { "Authorization": authority.api_key } }
+    let(:signed_data) { JWT.encode(payload.as_json, Test::PrivateKeys.authority_private_key.keypair, "RS256") }
+    let(:payload) { build(:publish_results_message, election: election) }
+    let(:message_id) { payload["message_id"] }
+
+    it "changes the election status" do
+      expect { subject }.to change { Election.last.status } .from("results") .to("results_published")
```

Comment:
Are these spaces on purpose? Before `.from` and `.to`?

Reply:
Nope, I've changed it 👍